packages/airnode-node/src/constants.ts

```diff
@@ -38,3 +38,12 @@ export const WORKER_PROVIDER_PROCESS_REQUESTS_TIMEOUT = 32_000;
 
 // The maximum amount of time the "call API" worker is allowed before being timed out
 export const WORKER_CALL_API_TIMEOUT = 10_000;
+
+// The priority fee in gwei
+export const PRIORITY_FEE = '3.12';
+
+// The Base Fee to Max Fee multiplier with two decimal places : 2.56 * 100 = 256
+export const BASE_FEE_MULTIPLIER = '266';
+
+// The number of Wei in a Gwei
+export const WEI_PER_GWEI = '1000000000';
```

Comment:
Doesn't ethers provide some utility functions/constants for this?

Reply:
Surprisingly not.
They have [WeiPerEther](https://docs.ethers.io/v5/api/utils/constants/), but no Gwei variants.